source/_components/light.httplight.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: page
+title: "HTTP Light"
+description: "Instructions on how to setup the HTTP light component within Home Assistant."
+date: 2018-7-07 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: httplight.png
+ha_category: Light
+ha_release: 0.73
+ha_iot_class: "Local Polling"
+---
+
+
+The `httplight` platform lets you control HTTP lights from within Home Assistant.
+
+To add HTTP Lights to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: httplight
+    name: Light
+    host: 10.1.1.1
+    type: color
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration